vcr/store.go

```diff
@@ -65,8 +65,11 @@ func (c *vcr) StoreCredential(credential vc.VerifiableCredential, validAt *time.
 }
 
 func (c *vcr) writeCredential(subject vc.VerifiableCredential) error {
-	// validation has made sure there's exactly one!
-	vcType := credential.ExtractTypes(subject)[0]
+	vcType := "VerifiableCredential"
+	customTypes := credential.ExtractTypes(subject)
```

Comment:
subject.Type is guaranteed to contain "VerifiableCredential", and at most 1 other type.